sql/core/src/test/resources/sql-tests/inputs/sum.sql

```diff
@@ -0,0 +1,32 @@
+-- sum
```

Comment:
shall we make it clear that this only test interval sum?

Reply:
or you can add tests to `group-by.sql`

Reply:
I can move them to `group-by.sql`